### DISCUSSION TITLE
Preview is fine, but exported PDF svgs disappeared?

### DISCUSSION THREAD

**Discussion by ghost** [typst/typst#1964] [Q&A]
code: ![image](https://github.com/typst/typst/assets/87360846/49fdd0e9-fa38-4aba-b6cf-dbad8d1c0469)

web preview: ![image](https://github.com/typst/typst/assets/87360846/938b2a1b-0afd-4bfc-9084-5a417d4d4fb2)

exported pdf: ![image](https://github.com/typst/typst/assets/87360846/eda849bc-6fd5-49a2-8e27-a49c6ba0cf86)

How to fix it? 
```
#grid(
  columns: (40fr,  10fr),
  rows: (60pt, auto),
  gutter: 3pt,
  cell(height: 100%)[ \
#grid(
  columns: (auto,auto,auto,auto),
  rows: (0pt),
  gutter: 3pt,
cell(height: 100%)[],
cell(height: 100%)[],
cell(height: 100%)[],
cell(height: 100%)[]
)],
cell(height: 130%)[#figure(
  image("xx.svg", width: 100%)
)]
)
```

**Comment by LaurenzV** [comment]
Could you send the SVG file as well?

**Reply by ghost** [reply]
![13](https://github.com/typst/typst/assets/87360846/d5b418f8-5dc1-4533-9bd9-e0b65a3296a7)
Here is the svg file, made in processon.io

**Comment by Enivex** [comment]
Can you post the svg?

@LaurenzV

**Reply by ghost** [reply]
You can check the svg I posted now  :)

**Comment by pepa65** [comment]
Can your pdf viewer display `13.svg` straight?

**Reply by ghost** [reply]
```
#figure(
  image("13.svg", width: 100%)
)
```
It can display in my browser pdf viewer or use web typist export using #figure without #grid.
I think there might be some mistakes with the syntax. Do there any nice exam templates?
